mxnet-predict.spec

```diff
@@ -1,7 +1,7 @@
-### RPM external mxnet-predict 1.2.0
+### RPM external mxnet-predict 1.2.1.mod1
 %define projectname mxnet-predict
 %define releasename apache-mxnet-src-%{realversion}-incubating
-Source: https://github.com/apache/incubator-mxnet/releases/download/%{realversion}/apache-mxnet-src-%{realversion}-incubating.tar.gz
+Source: https://hqu.web.cern.ch/hqu/tools/mxnet/apache-mxnet-src-%{realversion}-incubating.tar.gz
```

Comment:
hi- we should use a GitHub fork in the cms-sw area instead of a random (likely short lived) web address for this source

Reply:
makes sense.
@smuzaffar  do we have a fork for mxnet external under cms-sw ?

Reply:
The tarball I made is based on https://github.com/apache/incubator-mxnet/releases/tag/1.2.1 and the patch at https://github.com/hqucms/incubator-mxnet/commit/f31352f26b816d8a2c7080f6fc81377ff4924369

Reply:
I can't fork repos on cms-externals. If you fork it for me, I have the rest prepared.